x/gov/codec.proto

```diff
@@ -0,0 +1,93 @@
+syntax = "proto3";
+
+package gov;
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+import "github.com/iov-one/weave/coin/codec.proto";
+import "google/protobuf/timestamp.proto";
+
+// Electorate defines who may vote in an election. This same group can be used in many elections
+// and is stored for re-use
+message Electorate {
+    // Participants defines a list of all signatures that are allowed to participate in a vote
+    repeated Participant participants = 1 [(gogoproto.nullable) = false];
+}
+
+// Participant clubs together a signature with a weight. The greater the weight
+// the greater the power of a participant.
+message Participant {
+    bytes signature = 1 [(gogoproto.casttype) = "github.com/iov-one/weave.Address"];
+    uint32 power = 2 [(gogoproto.casttype) = "Weight"];
```

Comment:
What is the difference between weight an power? Why two different terms?

Reply:
Well spotted. This was copied from the [multisig](https://github.com/iov-one/weave/blob/master/x/multisig/codec.proto#L25) proto. I think we are a bit inconsistent with the use of power and weight now. I will change this to weight, now.
I have opened https://github.com/iov-one/weave/issues/453

Reply:
👍 the term weight is very clear. "Power" woud need much more context specific definition I think.